docs/conf.py

```diff
@@ -154,12 +151,25 @@
     "common_links.rst",
 ]
 
-
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = False
 
 default_role = "obj"
 
+# Customizations for make linkcheck using regular expressions
+linkcheck_allowed_redirects = {
+    r"https://.+\.(org|io|com)": r".+(org|io|com)/en/.+",  # to different doc branches
```

Comment:
Wouldn't this allow redirects to any website on the top level `.org`, `.io` or `.com` domains, rather than just different doc branches?

Reply:
Yeah, I had been wondering how to handle this. I couldn't find a way in the linkcheck configuration options to specify that the domain name be the same between the link and the redirect without giving every domain name.  My thinking has been that with the `/en/` in the second part, chances are the redirect is going to be for a docs page or otherwise somehow related to internationalization.  I'll make this a bit more specific...we'll catch more things links that need to be updated, with the caveat that we'll need to add some special cases in there.